content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -7,8 +7,7 @@ title: Debug Services
 ---
 
 {{% capture overview %}}
-An issue that comes up rather frequently for new installations of Kubernetes is
-that a `Service` is not working properly.  You've run your `Deployment` and
+An issue that comes up rather frequently for new installations of Kubernetes is that a `Service` is not working properly. You've run your `Deployment` and
 created a `Service`, but you get no response when you try to access it.
 This document will hopefully help you to figure out what's going wrong.
```

Comment:
why change?

Reply:
is just cosmetic and not have a line with a single world in it. I know and understand from UX there is no difference however it just _flows_ when you're in edit mode so to speak.

you agree ?

Reply:
@DanyC97 Consider two things:

1. Scope creep

As you say, the line change is cosmetic rather than falling within the original scope of the proposed changes. Isolation of concerns is a good thing to achieve here.

2. Consistency

There are other lines where you changed content but didn't also remove line breaks in sentences. If you're going to make cosmetic changes, I'd like to see them consistently applied, at least within the scope of the PR if not the whole document. (And again, scope creep.)

I think removing arbitrary line breaks is a helpful change; I just want to see it applied consistently as a separate PR. Make sense?

Reply:
ack, will stick with the initial scope and follow up with the cosmetics on a different PR